iib/web/models.py

```diff
@@ -852,16 +857,10 @@ def from_json(cls, kwargs, batch=None):
                 '"bundles" should be either an empty array or an array of non-empty strings'
             )
 
-        # Check if no bundles `from_index and `binary_image` are specified
-        # if no bundles and and no from index then a empty index will be created
-        # if no binary image and just a from_index then we are not updating anything and it would
-        # be a no-op
-        if not request_kwargs.get('bundles') and (
-            not request_kwargs.get('from_index') or not request_kwargs.get('binary_image')
-        ):
-            raise ValidationError(
-                '"from_index" and "binary_image" must be specified if no bundles are specified'
-            )
+        # Check if no bundles and `from_index is specified
+        # if no bundles and and no from index then an empty index will be created which is a no-op
```

Comment:
You missed this one :stuck_out_tongue: 
```suggestion
        # if no bundles and no from index then an empty index will be created which is a no-op
```

Reply:
lol, I forgot the migration file too :p

Thanks for pointing it out :)